library/src/test/java/io/constructor/core/ConstructorIoTrackingTest.kt

```diff
@@ -52,21 +52,21 @@ class ConstructorIoTest {
     fun trackAutocompleteSelect() {
         val mockResponse = MockResponse().setResponseCode(204)
         mockServer.enqueue(mockResponse)
-        val observer = ConstructorIo.trackAutocompleteSelect("titanic", "tit", "Search Suggestions").test()
+        val observer = ConstructorIo.trackAutocompleteSelect("titanic", "tit", "Search Suggestions", null, "2347874").test()
         observer.assertComplete()
         val request = mockServer.takeRequest()
-        val path = "/autocomplete/titanic/select?autocomplete_section=Search%20Suggestions&original_query=tit&tr=click&key=copper-key&i=wacko-the-guid&ui=player-three&s=67&c=cioand-1.3.0&_dt="
+        val path = "/autocomplete/titanic/select?autocomplete_section=Search%20Suggestions&original_query=tit&tr=click&result_id=2347874&key=copper-key&i=wacko-the-guid&ui=player-three&s=67&c=cioand-1.3.0&_dt="
```

Comment:
I suggest reverting all changes to the existing tests and adding a single new test per function with the additional parameters